core/src/test/scala/integration/kafka/api/SaslSslAdminClientIntegrationTest.scala

```diff
@@ -35,5 +44,77 @@ class SaslSslAdminClientIntegrationTest extends KafkaAdminClientIntegrationTest
     super.tearDown()
     closeSasl()
   }
-  
+
+  val ACL2 = new AclBinding(new Resource(ResourceType.TOPIC, "mytopic2"),
+    new AccessControlEntry("User:ANONYMOUS", "*", AclOperation.WRITE, AclPermissionType.ALLOW));
+  val ACL3 = new AclBinding(new Resource(ResourceType.TOPIC, "mytopic3"),
+    new AccessControlEntry("User:ANONYMOUS", "*", AclOperation.READ, AclPermissionType.ALLOW));
+  val ACL_UNKNOWN = new AclBinding(new Resource(ResourceType.TOPIC, "mytopic3"),
+    new AccessControlEntry("User:ANONYMOUS", "*", AclOperation.UNKNOWN, AclPermissionType.ALLOW));
+
+  /**
+    * Test that ACL operations are not possible when the authorizer is disabled.
+    * Also see {@link kafka.api.KafkaAdminClientSecureIntegrationTest} for tests of ACL operations
+    * when the authorizer is enabled.
+    */
+  @Test
+  override def testAclOperations(): Unit = {
+    client = AdminClient.create(createConfig())
+    assertEquals(0, client.describeAcls(AclBindingFilter.ANY).all().get().size())
+    val results = client.createAcls(List(ACL2, ACL3).asJava)
+    assertEquals(Set(ACL2, ACL3), results.results().keySet().asScala)
+    results.results().values().asScala.foreach(value => value.get)
+    val results2 = client.createAcls(List(ACL_UNKNOWN).asJava)
+    assertEquals(Set(ACL_UNKNOWN), results2.results().keySet().asScala)
+    assertFutureExceptionTypeEquals(results2.all(), classOf[InvalidRequestException])
+    val results3 = client.deleteAcls(List(ACL1.toFilter, ACL2.toFilter, ACL3.toFilter).asJava)
+    assertEquals(Set(ACL1.toFilter, ACL2.toFilter, ACL3.toFilter), results3.results().keySet().asScala)
+    assertEquals(0, results3.results.get(ACL1.toFilter).get.acls.size())
+    assertEquals(Set(ACL2), results3.results.get(ACL2.toFilter).get.acls.asScala.map(result => result.acl()).toSet)
+    assertEquals(Set(ACL3), results3.results.get(ACL3.toFilter).get.acls.asScala.map(result => result.acl()).toSet)
+    client.close()
+  }
+
+  def waitForDescribeAcls(client: AdminClient, filter: AclBindingFilter, acls: Set[AclBinding]): Unit = {
+    TestUtils.waitUntilTrue(() => {
+      val results = client.describeAcls(filter).all().get()
+      acls == results.asScala.toSet
+    }, "timed out waiting for ACLs")
+  }
+
+  @Test
+  def testAclOperations2(): Unit = {
```

Comment:
It would be good to explain how this differs from the other, `2` doesn't give it away.